docs/quickstart/assessment/src/main/webapp/AuthorApi.jsp

```diff
@@ -0,0 +1,33 @@
+<%@ page
+    import="com.learnosity.quickstart.AuthorApp"
+    language="java" contentType="text/html; charset=ISO-8859-1"
+    pageEncoding="ISO-8859-1"%>
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<%!
+    private AuthorApp app;
+    public void jspInit() {
+        app = new AuthorApp();
+    }
+    public void jspDestroy() {
+        app = null;
+    }
+%>
+<html>
+    <head><link rel="stylesheet" type="text/css" href="css/style.css"></head>
+    <body>
+        <h1>Standalone Assessment Example - Author API</h1>
+
+        <!-- Items API will render the assessment app into this div. -->
+        <div id="learnosity-author"></div>
+
+        <!-- Load the Author API library. -->
+        <script src="https://authorapi.learnosity.com/?latest"></script>
```

Comment:
@markus-liang-lrn In here, let's change this "latest" value to "latest-lts", because I really want to emphasise that clients should be selecting an LTS version.

Reply:
@TheRealEdDawson I guess , this will apply to all sdks ?